hack/tests/e2e-ansible.sh

```diff
@@ -42,7 +34,7 @@ operator_logs() {
     header_text "Getting events"
     kubectl get events
     header_text "Getting operator logs"
-    kubectl logs deployment/memcached-operator
+    kubectl logs deployment/osdk-controller-manager -c manager
```

Comment:
why it is no longer mencached?

Reply:
the deployment that is created by kustomize is called osdk-controller-manager, the old default was the name of the project